backend/level-2/exercise.spec.ts

```diff
@@ -12,7 +12,8 @@ const app = express();
  */
 
 app.get('/', (req: unknown, res: unknown) => {
-    // TODO: complete this function
+    const myRes = res as express.Response
```

Comment:
This is wrong. Think of another way to define this type without using `as`.

Tip: you can update everything except the tests